### ISSUE TITLE
Can I have the fumo database?

### ISSUE THREAD

**Issue by thatfrozenfrog** [Nosesisaid/fumo-API#25]
I'm finding fumo image api-s for my current project. Discovered this repo and I'm quite amazed by the size of your database. Can you send me those fumo ( I'm using Python, maybe I have to re-write this all again )

Torrent file is oke

**Comment by v1ctorio** [comment]
Of course, if I find them I will send you them, also someone recommended me a free S3 like provider so I'll also look if I can revive this project

**Comment by OugiFormula** [comment]
kinda reviving this but if you dont mind me asking why not use something like catbox to store the images?

**Comment by thatfrozenfrog** [comment]
No idea, but I eventually run through AmiAmi store and use their API to fetch all the fumos. Still open since @v1ctorio hasn't found the images yet :)

**Comment by v1ctorio** [comment]
Hi there, just finished my exams - I'm really happy to see that every once in a while someone ask me about this project (Did anyone really use it??? wth??). TBH, this project was just a simple app that returned a big ass JSON written by a middle schooler learning javascript. I don't have the images anymore (well they probably are encrypted in some random HD which encryption key I forgot).**This database had nothing interesting or valuable at all**. It really was just a bunch of non-attributed pictures of fumos with an ID assigned to each.

I will revive the project, up from the scratch and make it a proper API including featured "fumos", captions, source and that shortly. And host either on R2 or a slefhosted solution. Happy to see that this project has interest (even the slightliest) :)

**Comment by OugiFormula** [comment]
Tbh I wanted to use it on my discord bot that I'm working on lol I always search for a database of images or api before I make one myself

**Comment by v1ctorio** [comment]
That was literally my reason lmao. I was learning to make discord bot and my friend asked me to add a command that sends fumos. 

I told them "yeah there's 100% already an api that get fumos, this will be easy to implement" but surprisingly, none existed. That's why I created this ><

**Comment by OugiFormula** [comment]
Yeah touhou fans and specifically fumo enjoyers are wild that's why I was surprised only you made a fumo api

**Comment by thatfrozenfrog** [comment]
At the time of making the request, I remember there's a Chinese Touhou group which also does the same fumo api thing, but they're getting DDoS-ed, so I can't use their api. 
( I forgot it's name, sorry )

To be honest, the whole fumo thing I added to my bot is just one big json and that's it
https://pastebin.com/8236CMGH

**Comment by OugiFormula** [comment]
ok so I decided to create a database of fumo pictures that I can later on turn into an api if I have the time and I will be using community effort to build it, I'm right now working on a cog for my bot that will allow people to submit fumo pictures that I can add to the database that will be 100% running on services like imgur that are more likely to stay alive than a private cdn that way as long as I can.

if victorio allow me I will send here the link to the discord server this idea will take place on, keep in mind that for now its just a plan and I will work on this during the week.

**Comment by thatfrozenfrog** [comment]
@OugiFormula I'm interested. Can I join too?

**Comment by OugiFormula** [comment]
@thatfrozenfrog sure dm me on discord yukiokoito :)

**Comment by thatfrozenfrog** [comment]
@OugiFormula done.

**Comment by v1ctorio** [comment]
Yeah that's quite literally what [v1ctorio/fumo-bot](https://github.com/v1ctorio/Fumo-bot) was in it's moment, gl with your project I guess. I'll try to get this one back up asap
